### PR TITLE
Updated compile subtype part

### PR DESCRIPTION
**Pull request by runjin326**
### Purpose
Added a section describing `molecular-subtyping-pathology` section, which compiled subtypes, curated pathology diagnosis free text, and added clinical review. 

### Issue

https://github.com/AlexsLemonade/OpenPBTA-manuscript/issues/136

#### Which areas should receive a particularly close look?
I added a table to describe how we modify histology and diagnosis but if it's too detailed, we don't have to keep it and can just point to the repo. 

#### Is there anything that you want to discuss further?
No.

#### Is the pull request ready for review?
Yes.

### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [ ] All figures follow the [Manubot figure instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures)
- [ ] There are no new spelling errors identified by the [Manubot spellchecker](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#spellchecking)

### Spellcheck Step

The dictionary used for spellchecking can be updated.
Edit the file in `build/assets/custom-dictionary.txt` by adding new entries to the end.
You do not need to change anything else.
However, if you want to update the first line to have an accurate count of words and you want to remove non-unique ones, run the following command from within `build/assets` on your favorite OS X or Linux machine:
```
(( len = $(awk '!a[$0]++' < custom-dictionary.txt | wc -l ) - 1 )); tmpfile="$(mktemp)"; echo "personal_ws-1.1 en $len utf-8" > $tmpfile; tail -n +2 custom-dictionary.txt | awk '!a[$0]++' >> $tmpfile; mv $tmpfile custom-dictionary.txt
```

### CODE REVIEW

File: content/03.methods.md [hunk 1]
```diff
@@ -398,6 +398,55 @@ Craniopharyngiomas (CRANIO) were subtyped into adamantinomatous (`CRANIO, ADAM`)
...
+
+Firstly, `broad_histology`, `short_histology`, and `integrated_diagnosis` columns in the result files from the above subtyping modules (i.e., `CRANIO_molecular_subtype.tsv`, `EWS_results.tsv`, `EPN_all_data_withsubgroup.tsv`, `HGG_molecular_subtype.tsv` ,`lgat_subtyping.tsv`, `MB_molecular_subtype.tsv`, `embryonal_tumor_molecular_subtypes.tsv`, and `neurocytoma_subtyping.tsv`) were updated based on the molecular subtype of the tumor. 
+Detailed information about the updating procedure were included in the analysis [notebook](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-pathology/01-compile-subtyping-results.Rmd).
+Notes were also added to indicated that the changes in `broad_histology`, `short_histology` and `integrated_diagnosis` were from OpenPBTA subtyping modules.
+
+Subsequently, `broad_histology`, `short_histology` and `harmonized_diagnosis` columns of tumors with particular pathology diagnosis free text were updated as specified in the following table:
+pathology_diagnosis | subtyping module | pathology_free_text_diagnosis | broad_histology | short_histology | harmonized_diagnosis
```
Comment (jaclyn-taroni) [inline content/03.methods.md:409]:
```suggestion

pathology_diagnosis | subtyping module | pathology_free_text_diagnosis | broad_histology | short_histology | harmonized_diagnosis
```

File: content/03.methods.md [hunk 2]
```diff
@@ -398,6 +398,55 @@ Craniopharyngiomas (CRANIO) were subtyped into adamantinomatous (`CRANIO, ADAM`)
...
+Meningioma | NA | contains `atypical` | Meningioma | Meningioma | Atypical meningioma
+Meningioma | NA | contains `anaplastic` | Meningioma | Meningioma | Anaplastic (malignant) meningioma
+Meningioma | NA | contains `clear cell meningioma` | Meningioma | Meningioma | Clear cell meningioma
+Meningioma | NA | contains `meningothelial` | Meningioma | Meningioma | Meningothelial meningioma
+Meningioma | NA | does not contain `atypical`, `anaplastic`, `clear cell`, or `meningothelial` | Meningioma | Meningioma | Meningioma
+Choroid plexus papilloma | NA | contains `atypical` | Choroid plexus tumor | Choroid plexus tumor | Atypical choroid plexus papilloma
+Craniopharyngioma | CRANIO | contains `adamantinomatous` | Tumors of sellar region | Craniopharyngioma | Adamantinomatous craniopharyngioma 
+
+Similarly, `broad_histology`, `short_histology`, `integrated_diagnosis` and `harmonized_diagnosis` columns of tumors with following pathology diagnosis free text were updated as specified in the table below:
+pathology_diagnosis | subtyping module | pathology_free_text_diagnosis | broad_histology | short_histology | integrated_diagnosis
```
Comment (jaclyn-taroni) [inline content/03.methods.md:422]:
```suggestion

pathology_diagnosis | subtyping module | pathology_free_text_diagnosis | broad_histology | short_histology | integrated_diagnosis
```

**Review by jaclyn-taroni — COMMENTED** [review]
I'm going to commit the spacing changes that will allow the tables to be reviewed, so returning review early.

File: content/03.methods.md [hunk 3]
```diff
@@ -398,6 +398,57 @@ Craniopharyngiomas (CRANIO) were subtyped into adamantinomatous (`CRANIO, ADAM`)
...
+Low-grade glioma/astrocytoma (WHO grade I/II) | NA, remove from LGAT module | contains `glioneuronal` | Neuronal and mixed neuronal-glial tumor | GNT | Glial-neuronal tumor NOS, <subtype> | Glial-neuronal tumor NOS
+Low-grade glioma/astrocytoma (WHO grade I/II) | NA, remove from LGAT module | contains `rosette forming glioneuronal tumor` | Neuronal and mixed neuronal-glial tumor | GNT | Rosette-forming glioneuronal tumor, <subtype> | Rosette-forming glioneuronal tumor
+
+Notes were also added to indicated that the changes in `broad_histology`, `short_histology`, `integrated_diagnosis` and `harmonized_diagnosis` were from pathology diagnosis free text.
+
+For PNOC003 samples, `HGG` or `DMG` were assigned through HGG molecular subtyping module. 
+This may or may not be consistent with the molecular subtype assigned in clinical review. 
+For samples with subtype discrepancies, `molecular_subtype` and `integrated_diagnosis` were updated to reflect clinical review.
+Notes were also added to indicate updates were from clinical review.
```
Comment (jharenza) [inline content/03.methods.md:447]:
```suggestion
For samples with subtype discrepancies, `molecular_subtype` and `integrated_diagnosis` were updated following pathology or clinical review.
```

File: content/03.methods.md [hunk 4]
```diff
@@ -398,6 +398,57 @@ Craniopharyngiomas (CRANIO) were subtyped into adamantinomatous (`CRANIO, ADAM`)
...
+
+Notes were also added to indicated that the changes in `broad_histology`, `short_histology`, `integrated_diagnosis` and `harmonized_diagnosis` were from pathology diagnosis free text.
+
+For PNOC003 samples, `HGG` or `DMG` were assigned through HGG molecular subtyping module. 
+This may or may not be consistent with the molecular subtype assigned in clinical review. 
+For samples with subtype discrepancies, `molecular_subtype` and `integrated_diagnosis` were updated to reflect clinical review.
+Notes were also added to indicate updates were from clinical review. 
+Detailed information can be found in the analysis [notebook](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-pathology/03-incorporate-clinical-feedback.Rmd).
+
+Lastly, some samples were manually inspected by pathologists in CHOP and if discrepancies occur, `broad_histology`, `short_histology`, `integrated_diagnosis` and `molecular_subtype` were updated based on pathologist review.
```
Comment (jharenza) [inline content/03.methods.md:450]:
```suggestion
```

File: content/03.methods.md [hunk 5]
```diff
@@ -398,6 +398,57 @@ Craniopharyngiomas (CRANIO) were subtyped into adamantinomatous (`CRANIO, ADAM`)
...
+Detailed information can be found in analysis [notebook](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-pathology/04-incorporate-pathology-feedback.Rmd).
```
Comment (jharenza) [inline content/03.methods.md:451]:
```suggestion
```

File: content/03.methods.md [hunk 6]
```diff
@@ -398,6 +398,57 @@ Craniopharyngiomas (CRANIO) were subtyped into adamantinomatous (`CRANIO, ADAM`)
...
+Low-grade glioma/astrocytoma (WHO grade I/II) | NA, remove from LGAT module | contains `rosette forming glioneuronal tumor` | Neuronal and mixed neuronal-glial tumor | GNT | Rosette-forming glioneuronal tumor, <subtype> | Rosette-forming glioneuronal tumor
+
+Notes were also added to indicated that the changes in `broad_histology`, `short_histology`, `integrated_diagnosis` and `harmonized_diagnosis` were from pathology diagnosis free text.
+
+For PNOC003 samples, `HGG` or `DMG` were assigned through HGG molecular subtyping module. 
+This may or may not be consistent with the molecular subtype assigned in clinical review. 
+For samples with subtype discrepancies, `molecular_subtype` and `integrated_diagnosis` were updated to reflect clinical review.
+Notes were also added to indicate updates were from clinical review. 
+Detailed information can be found in the analysis [notebook](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-pathology/03-incorporate-clinical-feedback.Rmd).
```
Comment (jharenza) [inline content/03.methods.md:448]:
```suggestion
Detailed information can be found in the analysis notebooks for [clinical](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-pathology/03-incorporate-clinical-feedback.Rmd) and [pathology](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-pathology/04-incorporate-pathology-feedback.Rmd) feedback.
```

**Pull request merged**